### ISSUE TITLE
Regression Type & Dynamic vs. Static

### ISSUE THREAD

**Issue by mekazanc** [google/tfp-causalimpact#29]
Hello, I have a subtle question about the regression part. 

I assume that main code uses static regression and dynamic one is not included. 

Can you correct me if I am wrong ? 

Thanks in advance

**Comment by kylepl** [comment]
That is correct, it is using Static regression with SpikeAndSlab prior. See where the Gibbs model is [built](https://github.com/tensorflow/probability/blob/e0ffe4bb3e3792c2ca7cd4ba1c7f5fe135efd81c/tensorflow_probability/python/experimental/sts_gibbs/gibbs_sampler.py#L349). Notably, it does not mention [DynamicLinearRegression](https://github.com/tensorflow/probability/blob/e0ffe4bb3e3792c2ca7cd4ba1c7f5fe135efd81c/tensorflow_probability/python/sts/components/dynamic_regression.py), which, however, I'm not sure if even that is equivalent to the original CausalImpact implementation.
